src/jllvm/vm/Interpreter.hpp

```diff
@@ -210,6 +210,9 @@ class Interpreter : public OSRTarget
     /// Enable OSR from the interpreter into the JIT if the method is hot enough.
     bool m_enableOSR;
 
+    /// Single entry for use in 'm_interpreterCCSymbols' as an implementation for ALL methods.
+    std::uint64_t (*m_interpreterEntry)(const Method*, const std::uint64_t*){};
```

Comment:
This could be static

Reply:
This would needlessly restrict the whole program to only have a single `Interpreter` instance so I'd prefer not to.
Since the function is generated by LLVM it is specific to the current execution session (part of the `Runtime` class) and its lifetime also only restricted to the execution session.